### ISSUE TITLE
Tumblr links / Links to Tumblr accounts

### ISSUE THREAD

**Issue by njoy75** [Athou/commafeed#879]
Hi there,

Since the new RGDP European law, I can't have access to Tumblr accounts via CommaFeed like I used to before that.
Do you know if there is a special action I should do to follow these Tumblr feeds via CommaFeed ?
Thank you very much for your help.

**Comment by Athou** [comment]
Looks like I can subscribe to tumblr blogs just fine, what url are you using to subscribe?

**Comment by njoy75** [comment]
the original one : sort of : http://revuedecapage.tumblr.com/

**Comment by Athou** [comment]
Ah right, I can reproduce the issue.
Looks like an issue on Tumblr's side though, links like http://revuedecapage.tumblr.com/rss should not redirect to a GDPR warning page since it should be considered as an API call.

**Comment by njoy75** [comment]
So what should I do/change to the orginal tumblr link to keep on following them via CommaFeed?

**Comment by Athou** [comment]
Not much you can do except notify Tumblr about the issue. They have to change something on their side.

**Comment by njoy75** [comment]
ok, thx for your help and quick replies.

**Comment by njoy75** [comment]
Just to let you know that I got to Tumblr but they doesn't seem to reply.
In the meantime, I visited some forums where the topic has been discussed. The problem is about the way Tumblr is applying GDPR policy for European users. 
But I also discovered that some other aggregators as CommaFeed (Feedburner, Tiny Tiny RSS, The Old Reader....) keep reading the Tumblr feeds nontheless. So maybe CommaFeed could fix it so that it can keep reading Tumblr feeds like some of its competitors ?
